### PR TITLE
gui: add about page under help menu

### PR DESCRIPTION
**Pull request by IllusionMan1212**
Title says it all.
- Version string
- Short description about the emulator
- Useful links

here's how it looks btw
image

### CODE REVIEW

**Comment by wfscans** [comment]
May I suggest expanding the URL links? I assume those are clickable? Anyway... example:
[patreon.com/Vita3K](https://patreon.com/Vita3K)
\[patreon.com/Vita3K\](https://patreon.com/Vita3K)

**Comment by IllusionMan1212** [comment]
@wfscans they're actually buttons you just can't see them cause they're transparent

**Comment by wfscans** [comment]
Still, I'd suggest expanding them.

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    ImGui::Separator();
+
+    ImGui::Text("If you'd like to show your support to the project you can donate to our patreon");
+
+    std::ostringstream Link;
+
+    if (ImGui::Button("Patreon.com/Vita3K")) {
+        std::string Patreon = "https://patreon.com/vita3k";
+        Link << OsPrefix << Patreon;
+        system(Link.str().c_str());
```
[the remark this answers is not in the document]
Comment (VelocityRa) [reply src/emulator/gui/src/ui_about_dialog.cpp]: For unix systems you have to prepend the string with "`xdg-open `", else it won't work afaik.

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    ImGui::Text("\nVita3K: a PS Vita/PSTV Emulator. The world's first functional PS Vita/PSTV emulator");
+    ImGui::Text("Vita3K is an experimental open-source PlayStation Vita/PlayStation TV emulator\nwritten in C++ for Windows, MacOS and Linux operating systems.");
+    ImGui::Text("\nNote: the emulator is still in a very early stage of development.");
+
+    ImGui::Separator();
+
+    ImGui::Text("If you'd like to show your support to the project you can donate to our patreon");
+
+    std::ostringstream Link;
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    std::ostringstream link;
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    if (ImGui::Button("Patreon.com/Vita3K")) {
+        std::string Patreon = "https://patreon.com/vita3k";
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
        std::string patreon_url = "https://patreon.com/vita3k";
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#ifdef _WIN32
+static const char OsPrefix[] = "start ";
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]: `OS_PREFIX`

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+    std::ostringstream Link;
+
+    if (ImGui::Button("Patreon.com/Vita3K")) {
+        std::string Patreon = "https://patreon.com/vita3k";
+        Link << OsPrefix << Patreon;
+        system(Link.str().c_str());
+    }
+
+    ImGui::Text("Or if you're interested in helping develop the emulator check out the github");
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    ImGui::Text("Or if you're interested in contributing, check out our GitHub");
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+    if (ImGui::Button("Github.com/Vita3K/Vita3K")) {
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    if (ImGui::Button("github.com/Vita3K/Vita3K")) {
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+        std::string Github = "https://github.com/vita3k/vita3k";
+        Link << OsPrefix << Github;
+        system(Link.str().c_str());
+    }
+
+    ImGui::Text("Visit our website for more info");
+
+    if (ImGui::Button("Vita3K.org")) {
+        std::string Website = "https://vita3k.org";
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
        std::string website_url = "https://vita3k.org";
```

File: src/emulator/gui/src/ui_main.cpp [hunk 8]
```diff
@@ -119,5 +119,8 @@ void DrawUI(HostState &host) {
     if (host.gui.allocations_dialog) {
         DrawAllocationsDialog(host);
     }
+	if (host.gui.about_dialog) {
+        DrawAboutDialog(host);
+	}
```
Comment (1whatleytay) [inline src/emulator/gui/src/ui_main.cpp]: Don't mind me if I'm being a little too picky but... isn't this indentation a little weird?

Comment (IllusionMan1212) [reply src/emulator/gui/src/ui_main.cpp]: being picky is fine, and i don't really know what happened there lol

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+    ImGui::Separator();
+
+    ImGui::Text("\nVita3K: a PS Vita/PSTV Emulator. The world's first functional PS Vita/PSTV emulator");
+    ImGui::Text("Vita3K is an experimental open-source PlayStation Vita/PlayStation TV emulator\nwritten in C++ for Windows, MacOS and Linux operating systems.");
+    ImGui::Text("\nNote: the emulator is still in a very early stage of development.");
+
+    ImGui::Separator();
+
+    ImGui::Text("If you'd like to show your support to the project you can donate to our patreon");
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    ImGui::Text("If you'd like to show your support to the project, you can donate to our Patreon:");
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+    ImGui::SetNextWindowSize(ImVec2(width, height));
+    ImGui::Begin("About", &host.gui.about_dialog);
+
+    ImGui::Text(window_title);
+
+    ImGui::Separator();
+
+    ImGui::Text("\nVita3K: a PS Vita/PSTV Emulator. The world's first functional PS Vita/PSTV emulator");
+    ImGui::Text("Vita3K is an experimental open-source PlayStation Vita/PlayStation TV emulator\nwritten in C++ for Windows, MacOS and Linux operating systems.");
+    ImGui::Text("\nNote: the emulator is still in a very early stage of development.");
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    ImGui::Text("\nNote: The emulator is still in a very early stage of development.");
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+    float height = ImGui::GetWindowHeight() / 1.35;
+
+    ImGui::SetNextWindowSize(ImVec2(width, height));
+    ImGui::Begin("About", &host.gui.about_dialog);
+
+    ImGui::Text(window_title);
+
+    ImGui::Separator();
+
+    ImGui::Text("\nVita3K: a PS Vita/PSTV Emulator. The world's first functional PS Vita/PSTV emulator");
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    ImGui::Text("\nVita3K: a PS Vita/PSTV Emulator. The world's first functional PSVita/PSTV emulator");
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+    ImGui::Text("Or if you're interested in contributing, check out our Github");
+
+    if (ImGui::Button("github.com/vita3k/vita3k")) {
+        std::string github_url = "https://github.com/vita3k/vita3k";
+        link << OS_PREFIX << github_url;
+        system(link.str().c_str());
+    }
+
+    ImGui::Text("Visit our website for more info");
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    ImGui::Text("Visit our website for more info:");
```

File: src/emulator/gui/src/ui_about_dialog.cpp [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+
+    std::ostringstream link;
+
+    if (ImGui::Button("patreon.com/vita3k")) {
+        std::string patreon_url = "https://patreon.com/vita3k";
+        link << OS_PREFIX << patreon_url;
+        system(link.str().c_str());
+    }
+
+    ImGui::Text("Or if you're interested in contributing, check out our Github");
```
Comment (VelocityRa) [inline src/emulator/gui/src/ui_about_dialog.cpp]:
```suggestion
    ImGui::Text("Or if you're interested in contributing, check out our Github:");
```

**Pull request closed without merging**